src/librustc_unicode/char.rs

```diff
@@ -362,7 +362,7 @@ impl char {
     /// mostly similar to ID_Start but modified for closure under NFKx.
     #[unstable(feature = "unicode",
                reason = "mainly needed for compiler internals",
-               issue = "0")]
+               issue = "27784")]
```

Comment:
Yeah this is a very old feature name and nowadays it'd be named something different, but alas!